docs/coding_style.rst

```diff
@@ -0,0 +1,116 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+============
+Coding Style
+============
+
+Code you deliver to Mozilla as a consultant should be written according to
+the same standards as code written by any other Mozilla developer. Plan for
+your code to be maintained by other coders and comment liberally.
+
+Standardizing a common style for writing and formatting code helps a team
+or organization maintain a sensible code base. The goal is consistency and
+predictability so that any coder can view the work of another coder and
+quickly follow along. We have `documented style guides`_ elsewhere that go
+into greater depth, but we'll also summarize the main points here for
+some common languages.
+
+.. _`documented style guides`: http://mozweb.readthedocs.org/latest/reference
+
+.. Important::
+
+  We urge you to read through the `Mozilla Webdev Bootcamp`_ docs for more 
+  detailed guidelines.
+
+.. _`Mozilla Webdev Bootcamp`: http://mozweb.readthedocs.org/
+
+
+General
+~~~~~~~
+
+* Use spaces for indentation. Never use tabs – history has shown that
+  we cannot handle them.
+* Use four spaces to indent most languages, but two spaces in HTML. HTML lends
+  itself to a lot of nested elements and indenting each level four spaces can
+  lead to long lines and messy formatting.
+* Wrap lines at 80 characters when possible. HTML is often the exception here
+  as well.
+* Eliminate trailing whitespace at the end of lines. Blank lines should have
+  no spaces.
+* Include a single blank line at the end of files.
+
+.. Note::
+
+  Using EditorConfig_ in your project can help enforce most of the above 
```

Comment:
Do we have a default .editorconfig file we can link too?

Reply:
Not yet, but this might be something worth adding in later. For now, I think linking to bedrock's file may be a good enough example.